modules/orbit/pom.xml

```diff
@@ -66,6 +66,29 @@
             <artifactId>bcprov-jdk15on</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>opensaml1</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>opensaml</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>xmltooling</artifactId>
+            <version>${opensaml.xmltooling.version}</version>
```

Comment:
can we add these versions to the dependency management section in the root pom of the project and get rid of them from here?

Reply:
Yeah i tried that too but when you add to the root pom then, tests are failing

Reply:
This was fixed by remove xmltooling and openws dependencies